City-of-Helsinki/drupal-helfi-platform-config
# UHF-1173: Matomo config

How to test:

- Clone a new environment
- Checkout this branch of the module: `composer require drupal/helfi_platform_config:dev-UHF-1173_matomo_config`
- Build the site: `make new`
- Log in as admin and make sure that
  - `Matomo` and `Matomo reports` modules are enabled
  - Both modules have configuration in place
  - Both modules have permissions set (mostly for administrator role)
